Bank_Account.rb

```diff
@@ -79,6 +79,16 @@ def self.find(id) #returns an instance of Account where the value of the id fiel
         end
         puts "\n#{idmatch}"
         puts idmatch.balance
```

Comment:
When we use variables with more than one "word" we should separate those with the `_` so this should be `id_match`